403 Forbidden Errors when downloading dependencies

Hi there,

I am using httpX action in my workflow and as it is downloading the dependencies I receive the following error:
```
go: downloading github.com/lucasb-eyer/go-colorful v1.2.0
go: downloading github.com/rivo/uniseg v0.4.4
go: downloading github.com/tidwall/pretty v1.2.1
go: downloading github.com/dlclark/regexp2 v1.8.1
go: downloading github.com/tidwall/tinyqueue v0.1.1
Error: ../../../../go/pkg/mod/github.com/mholt/archiver@v3.1.1+incompatible/lz4.go:8:2: github.com/pierrec/lz4@v2.6.1+incompatible: reading https://proxy.golang.org/github.com/pierrec/lz4/@v/v2.6.1+incompatible.zip: 403 Forbidden
Error: ../../../../go/pkg/mod/github.com/refraction-networking/utls@v1.5.2/u_handshake_client.go:15:2: github.com/klauspost/compress@v1.16.7: reading https://proxy.golang.org/github.com/klauspost/compress/@v/v1.16.7.zip: 403 Forbidden
Error: Process completed with exit code 1. 
```

In this particular example my input list contains 1000 hosts however I've received this error for other scans too. Below is my config however I don't think this will help as its failing even before the scan starts:
```
      - name: httpX Scan
        uses: projectdiscovery/httpx-action@main
        with:
          list: .github/input/input.txt
          output: ~/output.txt
          flags: "-config .github/configurations/httpx/httpx-config.yaml"
```

Any ideas why this is the case?

Many thanks

Rishi

--- a/action.yml
+++ b/action.yml
@@ -41,6 +41,8 @@ runs:
         ${{ inputs.random-ua }} && echo "random-ua=-random-agent" >> $GITHUB_ENV
         echo "/home/runner/go/bin/" >> $GITHUB_PATH
       shell: bash
+      env:
+        GOPROXY: https://proxy.golang.org,direct
 
     - run: |
         httpx \

